### ISSUE TITLE
Fix Scrutinizer for Skosmos

### ISSUE THREAD

**Issue by joelit** [NatLibFi/Skosmos#1734]
## URL address of the page where you encountered the problem
For example - https://scrutinizer-ci.com/g/NatLibFi/Skosmos/inspections/3a4dd9b1-f7d8-4449-93a7-42b29f7e0315

## Description of the problem
Recently the scrutinizer tested stopped working. The failure seems to be:
```
> node index.js --exec install

/home/scrutinizer/build/node_modules/tough-cookie/dist/memstore.js:30
        const result = this.idx[domain]?.[path]?.[key];
                                        ^

SyntaxError: Unexpected token .
```

## Additional information (e.g. screenshots) about the problem

This can probably be configured at https://scrutinizer-ci.com/g/NatLibFi/Skosmos/ with sufficient  access rights. One possibility is that Node.js is at old version.

**Comment by osma** [comment]
Yes, it appears to be a problem where Scrutinizer by default installs Node.js 10, an ancient version.

Our current GitHub Actions [CI pipeline](https://github.com/NatLibFi/Skosmos/blob/main/.github/workflows/ci.yml) uses Node.js 16, but it's nowadays EOL, so I decided to try Node.js 18 first by using this Scrutinizer configuration:

```
build:
  image: default-bionic
  environment:
    php: 8.1.2
    node: v18.20.6
```

However, that didn't work. Builds now fail with this error:

```
node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by node)
nvm is not compatible with the npm config "prefix" option: currently set to ""
Run `nvm use --delete-prefix v18.20.6` to unset it.
```

It appears that Node.js 18 is not compatible with the Ubuntu Bionic (18.04 LTS) base image that Scrutinizer uses (its glibc version is too old). We used `image: default-bionic` as advised [here](https://scrutinizer-ci.com/docs/guides/upgrading-trusty-image). However, the `image` setting appears to be otherwise undocumented and I can't find a list of valid values.

I removed the `image: default-bionic` setting, but it didn't seem to make a difference; installing Node.js 18 still fails with the above GLIBC_2.28 error. So I switched to `node: v16.20.2` instead. It seems to work!

**Comment by joelit** [comment]
Sounds good! The v 16.20.2 sounds oddly specific. This might be a thing where we just have to wait it out for a while before updaing Node.js to a newer version. Maybe checking back to this in a year might be a good idea.

**Comment by osma** [comment]
Scrutinizer config requires a specific version number. v16.20.2 is the last release in the v16 series, and very likely the final one, since v16 is EOL.
